packages/rest/test/integration/http-handler.integration.ts

```diff
@@ -58,6 +58,12 @@ describe('HttpHandler', () => {
         .withOperationReturningString('get', '/hello', 'hello')
         .withOperationReturningString('get', '/bye', 'bye')
         .withOperationReturningString('post', '/hello', 'postHello')
+        .withOperationReturningStringWithStatusCode(
```

Comment:
Not needed for this spike, but required for the real implementation:

Please create a new `describe` block the 201 scenario. This set of tests is focused on testing different paths and verbs, not status codes.